---
title: Genetics of common complex psychiatric disorders
author: Mark Adams
format: 
  revealjs: default
  pdf: default
  gfm: default
--- 

Mark Adams  
Division of Psychiatry  
`mark.adams@ed.ac.uk`  
*Genetics and Environmental Influences on Behaviour and Mental Health*

# Topics:

- **Quantitative genetics and heritability**
- **Candidate gene studies**
- **Genome-wide studies**
- **Prediction**
- **Causality**

# What is a *"common"*, *"complex"* psychiatric disorder?

**Common:** Affects 1% or more of the population  
**Complex:** Inheritance cannot be explained by a single gene  

::: {.notes}
Psychiatric disorders are defined by disruption to higher-order brain functions of moods, perceptions, thoughts, beliefs, and behaviours but usually in the absence of major neurological impairments (consciousness, senses, memory). Psychiatric disorders include depressive and anxiety disorders (major depressive disorder, panic disorder), manic and psychotic disorders (bipolar disorder, schizophrenia), obsessive-compulsive disorders, eating disorders (anorexia nervosa, bulimia nervosa), substance-use disorders and personality disorders. Childhood conditions like attention-deficit/hyperactivity and autism can also be included, but only when they lead to clinically-salient impairment or distress. There are also many shades of sadness, hallucinations, eccentricities, mood swings, body-image preoccupations, recreational substances use, personalities, etc that are not psychiatric disorders but may still be informative to study from an aetiological and genetic standpoint.

- Sullivan PF and Geschwind DH (2019) [Defining the Genetic, Genomic, Cellular, and Diagnostic Architectures of Psychiatric Disorders](https://doi.org/10.1016/j.cell.2019.01.015). *Cell* doi:10.1016/j.cell.2019.01.015

:::

## üß¨üë™üö¨üí¢üè°üíûü©ªüè´

- Depression: 3% in a week
- Schizophrenia: 1% in lifetime
- Bipolar disorder: 2% in lifetime
- Anxiety disorder: 6% in a week

::: {.notes}
Psychiatric disorders have many causes, correlates, and consequences (genetics, environment, family life, substance use, relationships)

Incidence of psychiatric disorders range from the common (depression, anxiety) to the rare (bipolar disorder, schizophrenia). 
:::

# Why genetics?

Why use genetics to study mental health and psychiatric disorders?

- Biological understanding of genes, pathways
- Shared aetiology with other disorders
- Risk prediction
- Drug retargeting
- Causal analysis of environmental risk factors

## Genetics of categorical traits

![Diagram showing the seven "characters" observed by Mendel](assets/Mendel_seven_characters.svg)

::: {.notes}
Gregor Mendel (1822‚Äì1884), working in what is now Czechia, discovered the transmission of traits from parents to offspring could be explained by the inheritance of two "elements", which we now call alleles. Mendel was concerned with discrete or categorical phenotypes.

[Mendel pea plant figure](https://commons.wikimedia.org/wiki/File:Mendel_seven_characters.svg) by Mariana Ruiz (LadyofHats) [public domain]
:::

## Genetics of continuous traits

![](assets/Galton-Regression-PlateIX.png)

::: {.notes}
Separately, Francis Galton (1822‚Äì1911), was studying the inheritance of continuous or metric phenotypes. He noticed the parents who were tall tended to have children that were slightly shorter than themselves (and vice versa). This was termed "regression to the mean" from which the name of the statistical method "regression" is derived. 

> "In their search for universal hereditary laws, Galton and Pearson were driven by the linear model and the normal distribution because the associated parameters had scientific meaning for them that went beyond mere description."
- Wachsmuth, A., Wilkinson, L., & Dallal, G. E. (2003). [Galton's Bend](https://dx.doi.org/10.1198/0003130031874). The American Statistician, 57(3), 190‚Äì192. doi:10.1198/0003130031874

For more on Galton's legacy, see [https://adelphigenetics.org/history/](https://adelphigenetics.org/history/)
:::

## Reconciling categorical + continuous genetics = quantitative genetics

```{r polygenic_quantitative}
require(ggplot2)
require(dplyr)
require(stringr)

# calculate expected genotype frequency for number of increasing alleles
# use the density of the binomial distribution, for alleles = 0 up to 2 * number of loci. 
# assume each allele has the same frequency of 50% (for simplicity)
n_allele_freq <- function(n_loci) {
  alleles <- seq(from = 0, to = 2*n_loci)
  freq <- dbinom(alleles, size = 2*n_loci, prob = 0.5)
  data.frame(alleles, freq, loci = n_loci)
}

number_of_loci <- c(1, 2, 3, 10)
loci_freq <- bind_rows(lapply(number_of_loci, n_allele_freq))

loci_labeller <- function(string) {
  n_loci <- as.numeric(string)
  if_else(
    n_loci == 1, 
    true = "1 locus",
    false = str_c(string, "loci", sep = " "))
}
plot_loci <- function(loci_freq) {
  ggplot(loci_freq, aes(x = alleles, y = freq)) + geom_col() +
  facet_grid(
    . ~ loci,
    scales = "free_x",
    space = "free_x",
    labeller = labeller(loci = loci_labeller)
  ) +
  scale_x_continuous("Number of phenotype increasing alleles") +
  scale_y_continuous("Genotype frequency") +
  theme_linedraw()
}

plot_loci(
  filter(loci_freq, loci == 1)
)
```

<!-- ![](assets/Fisher-Supposition.png) -->
:::{.notes}
Ronald Fisher reconciled the inheritance of continuous and categorical phenotypes by showing that a continuous phenotype could be made from the inheritance of a large number (dozens, hundreds, or thousands) categorical genes. The term "variance" comes from Fisher's discoveries.

- Fisher, R. A. (1918). [XV.‚ÄîThe correlation between relatives on the supposition of Mendelian inheritance](https://dx.doi.org/10.1017/S0080456800012163). *Transactions of the Royal Society of Edinburgh*. doi:10.1017/S0080456800012163
- Charlesworth and Edwards (2018). [A century of variance](https://doi.org/10.1111/j.1740-9713.2018.01170.x). *Significance* 15(4). doi:10.1111/j.1740-9713.2018.01170.x
- Bodmer et al (2021) [The outstanding scientist, R.A. Fisher: his views on eugenics and race](https://dx.doi.org/10.1038/s41437-020-00394-6). _Heredity_ doi:10.1038/s41437-020-00394-6
:::

## Polygenic traits are quantitative traits

Adding up effects from a large number of genetic effects to make a continuous phenotype is related to the Central Limit Theorem.

```{r polygenic_quantitative_10}
plot_loci(loci_freq)
```

:::{.notes}
> "R. A. Fisher's 1918 paper, 'The correlation between relatives on the supposition of Mendelian inheritance', resolved the often bitter conflict between biometricians and Mendelians, which raged for a decade following the rediscovery of Mendel's work. Fisher showed that a complex quantitative trait could be explained by Mendelian inheritance if several genes affect the trait.
> "Because he crossed true-breeding plants, Mendel's experiments showed that a single locus with two alleles of equal frequency results in three genotypes (see the figure, part a). If the allelic effects are additive, the three genotypes produce three phenotypes; in the case of Mendel's qualitative traits, the allelic effects showed complete dominance, so only two phenotypes were observed. However, assuming equal and additive effects, 2 genes yield 9 genotypes and 5 phenotypes (part b) and 3 genes yield 27 genotypes and 7 phenotypes (part c). With unequal and non-additive allelic effects and some environmental influence, three genes would result in a normal bell-shaped curve of continuous variation (part d). This logic assumes common alleles; rare alleles will skew the distribution. Genome-wide association research suggests that many more than three genes affect most traits, which underscores the expectation that polygenic traits are quantitative traits."

- Plomin, R., Haworth, C. & Davis, O. [Common disorders are quantitative traits](https://dx.doi.org/10.1038/nrg2670). *Nat Rev Genet* **10**, 872-878 (2009). doi:10.1038/nrg2670
:::

# Biometrics

## What are the sources of family resemblance? How do we quantify them numerically?

## Heritability

Proportion of similarity in phenotypes that can be attributed to similarity in genotypes. 

**Model:** Phenotype (P) = Genotype (G) + Environment (E)  
**Variance decomposition**
$$var(P) = var(ùê∫) + var(ùê∏)$$
**Proportion of variance**
$$h^2 = \frac{var(ùê∫)}{var(ùëÉ)}$$

:::{.notes}
- Tenesa, A., Haley, C. [The heritability of human disease: estimation, uses and abuses](https://dx.doi.org/10.1038/nrg3377). _Nat Rev Genet_ **14**, 139‚Äì149 (2013). doi:10.1038/nrg3377
- Visscher, P., Hill, W. & Wray, N. [Heritability in the genomics era ‚Äî concepts and misconceptions](https://dx.doi.org/10.1038/nrg2322). _Nat Rev Genet_ **9**, 255‚Äì266 (2008). doi:10.1038/nrg2322
:::

## How to estimate heritability from data

```{r}
#| label: height
#| echo: false
require(readr)
require(dplyr)
require(ggplot2)

height_data <- read_tsv(here::here("data/galton-stata11/galton-stata11.tab"))

# filter to one offspring per family
# convert to cm
# average parental height
height_singletons <- height_data |>
  mutate(
    father_cm = father * 2.54,
    mother_cm = mother * 2.54,
    child_cm = height * 2.54
  ) |>
  mutate(parents_cm = (father_cm + mother_cm) / 2) |>
  group_by(family) |>
  slice_head(n = 1)
```

:::: {.columns}

::: {.column width="60%"}
```{r}
#| label: height_parent_offspring
#| fig-width: 5
#| fig-height: 5
require(ggplot2)

ggplot(height_singletons, aes(x = parents_cm, y = child_cm)) +
  geom_point() +
  scale_x_continuous("Parental height [avg.] (cm)") +
  scale_y_continuous("Child height (cm)")
```
:::

::: {.column width="40%"}
Plot of child (offspring) height versus the average of their parents' heights. What is a statistic that can be used to summarise the relationship between these two variables?
:::

::::



